.gitignore

```diff
@@ -1,11 +1,14 @@
-/build/
+/build/assembly/
+/build/descriptor/
+/build/build_logs/
 workers/unity/build/
 /logs/
 .spatialos/
 .idea/
 .vs/
 .DS_Store
 
+
```

Comment:
Was this intentional?

Reply:
Not really, but I'll leave it to save a CI round-trip